config.md

```diff
@@ -340,18 +340,38 @@ For Linux-based systems, the `process` object supports the following process-spe
 
     * **`class`** (string, REQUIRED) specifies the I/O scheduling class. Possible values are `IOPRIO_CLASS_RT`, `IOPRIO_CLASS_BE`, and `IOPRIO_CLASS_IDLE`.
     * **`priority`** (int, REQUIRED) specifies the priority level within the class. The value should be an integer ranging from 0 (highest) to 7 (lowest).
-* **`execCPUAffinity`** (object, OPTIONAL) specifies CPU affinity used to execute the process.
+* **`CPUAffinity`** (object, OPTIONAL) specifies CPU affinity used to execute processes in the runtime and the container.
+    All the properties in this setting expects as argument a string that contain a comma-separated list, with dashes to represent
+    ranges. For example, `0-3,7` represents CPUs 0,1,2,3, and 7. If omitted or empty for particular [lifecycle](runtime.md#lifecycle) stage,
+    the runtime SHOULD NOT change process' CPU affinity, and the affinity is determined by the Linux kernel.
```

Comment:
I think it would be very useful to have an `all` special value which indicates that the CPU affinity should be reset to include as many CPUs as possible.